### PR TITLE
Chapter 3

### PR DESCRIPTION
**Pull request by Minhhoang112331**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
```
Comment (tranquanghuy-rightsvn) [inline config/routes.rb:2]: remove this comment

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+  root 'static_pages#home'
```
Comment (tranquanghuy-rightsvn) [inline config/routes.rb:3]: use " istead of '

File: app/views/static_pages/home.html.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<% provide(:title, "Home") %>
+<h1>Sample App</h1>
+<p>
+  This is the home page for the
+  <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
+  sample application.
+</p>
```
Comment (tranquanghuy-rightsvn) [inline app/views/static_pages/home.html.erb:7]: add a empty line

File: app/views/static_pages/help.html.erb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<% provide(:title, "Help") %>
+<h1>Help</h1>
+<p>
+  Get help on the Ruby on Rails Tutorial at the
+  <a href="http://www.railstutorial.org/#help">Rails Tutorial help section</a>.
+  To get help on this sample app, see the
+  <a href="http://www.railstutorial.org/book"><em>Ruby on Rails Tutorial</em>
+  book</a>.
+</p>
```
Comment (tranquanghuy-rightsvn) [inline app/views/static_pages/help.html.erb:9]: add a empty line

File: app/controllers/static_pages_controller.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+class StaticPagesController < ApplicationController
+	def home
```
Comment (tranquanghuy-rightsvn) [inline app/controllers/static_pages_controller.rb:2]: one line

File: app/controllers/static_pages_controller.rb [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+	end
+
+	def help
```
Comment (tranquanghuy-rightsvn) [inline app/controllers/static_pages_controller.rb:5]: same as

File: config/routes.rb (hunk 2 above)
Comment (Minhhoang112331) [reply config/routes.rb:3]: istead of change resources ?

**Pull request merged**